cmd/conduit/root/connectors/list_test.go

```diff
@@ -54,3 +62,136 @@ func TestConnectorsListCommandFlags(t *testing.T) {
 		is.Equal(cf.Usage, f.usage)
 	}
 }
+
+func TestListCommandExecuteWithClient_NoFlags(t *testing.T) {
+	is := is.New(t)
+
+	buf := new(bytes.Buffer)
+	out := &ecdysis.DefaultOutput{}
+	out.Output(buf, nil)
+
+	ctx := context.Background()
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockService := mock.NewMockConnectorService(ctrl)
+
+	connectors := []*apiv1.Connector{
+		{
+			Id:           "conn1",
+			Type:         apiv1.Connector_TYPE_SOURCE,
+			Plugin:       "plugin1",
+			ProcessorIds: []string{"proc3"},
+			PipelineId:   "pipeline1",
+			CreatedAt:    testutils.GetDateTime(),
+			UpdatedAt:    testutils.GetDateTime(),
+		},
+		{
+			Id:         "conn2",
+			Type:       apiv1.Connector_TYPE_DESTINATION,
+			Plugin:     "plugin2",
+			PipelineId: "pipeline2",
+			CreatedAt:  testutils.GetDateTime(),
+			UpdatedAt:  testutils.GetDateTime(),
+		},
+	}
+
+	testutils.MockGetConnectors(mockService, "", connectors)
+
+	client := &api.Client{
+		ConnectorServiceClient: mockService,
+	}
+
+	cmd := &ListCommand{}
+	cmd.Output(out)
+
+	err := cmd.ExecuteWithClient(ctx, client)
+	is.NoErr(err)
+
+	output := buf.String()
+	is.True(len(output) > 0)
+
+	strings.Contains(output, ""+
```

Comment:
Should we actually test if output is equal to the expected string?

Reply:
@hariso tried that, but then I had to worried about possible subtle differences of a single space 🤦

Reply:
Figured out what's the actual difference, so I think I'll change them all. Thanks for the suggestion of taking another look at this.

Reply:
https://github.com/ConduitIO/conduit/pull/2111/commits/e091a228748c6284fbaea1cf34ef5d59b5607887